app/assets/stylesheets/_date-picker.scss

```diff
@@ -0,0 +1,30 @@
+.react-datepicker {
```

Comment:
@talyssonoc I had applied this solution https://github.com/Hacker0x01/react-datepicker/issues/624#issuecomment-261696513 to made it work.